### ISSUE TITLE
Cannot change params w/ VoiceOver?

### ISSUE THREAD

**Issue by hellomatty** [AudioKit/AudioKitSynthOne#160]
With VoiceOver enabled I can change the waveforms for Oscars 1 and 2 by swiping up or down with a single finger. This is not possible with other params, such as Oscars 1 semitones. Swiping up or down on those yields a negative audio cue and no param change.

I noticed that while the accessibility increment and decrement callbacks are implemented in the Stepper class, they are empty in the Knob class. I do not know if this is by design?

Is there an alternative VoiceOver interaction for changing params such as Oscars 1 semitones?

**Comment by hellomatty** [comment]
Ah, I get it. Single finger double tap and hold. Slide left or right to change the value. There is no audio feedback.

I'm guessing this is the direct interaction model declared in the knob's accessibility traits.
